requirements-dev.txt

```diff
@@ -1,7 +1,10 @@
 Cython==0.29.24 ; python_version <= '3.9'
-Cython==0.29.25 ; python_version >= '3.10'
+Cython==0.29.25 ; python_version == '3.10'
+Cython==0.29.30 ; python_version == '3.11'
 Jinja2==3.0.3
 numpy==1.19.5 ; python_version <= '3.9'
-numpy==1.21.6 ; python_version >= '3.10'
-pybind11==2.8.0
+numpy==1.21.6 ; python_version == '3.10'
+numpy==1.23.5 ; python_version == '3.11'

```

Comment:
>=  for 3.11 ?

Reply:
I'm not sure it's meaningful because numpy-1.23.5 highly likely will not support higher versions of python

Reply:
yes. but at least it will give it a chance if it would.